include/sampling/sampling.hpp

```diff
@@ -239,65 +217,48 @@ void logconcave_sampling(PointList &randPoints,
                          const Point &starting_point,
                          unsigned int const& nburns,
                          NegativeGradientFunctor &F,
-                         NegativeLogprobFunctor &f)
+                         NegativeLogprobFunctor &f) 
 {
-    typedef typename WalkTypePolicy::template Walk
-            <
-                    Point,
-                    Polytope,
-                    RandomNumberGenerator,
-                    NegativeGradientFunctor,
-                    NegativeLogprobFunctor,
-                    Solver
-            > walk;
-
-    typedef typename WalkTypePolicy::template parameters
-            <
-                    NT,
-                    NegativeGradientFunctor
-            > walk_params;
-
-    // Initialize random walk parameters
-    unsigned int dim = starting_point.dimension();
-    walk_params params(F, dim);
```

Comment:
how you pass the parameters to the walk?

Reply:
@vfisikop Thank you for pointing this out! So previously I was only relying on the assumption that the walk object will auto set the params. However after looking into how the langevin_walk is implemented, I saw that this is not the case. Thus, now I am setting the params using the set_params method. Let me know if this makes sense.